lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/NettyMessagingTransport.java

```diff
@@ -165,7 +165,7 @@ private NettyMessagingTransport(
         InetSocketAddress socketAddr = null;
         Channel acceptorFound = null;
         for (int port : tcpPortProvider) {
-          LOG.log(Level.FINEST, "Try port {0}", port);
```

Comment:
>FINEST [](start = 24, length = 6)

Change back to FINEST before sending pull request to apache/master. I left it here as it is helps me in debugging the containers feature.